### DISCUSSION TITLE
我可以使用你项目里生成 x-bogus 参数的接口吗？

### DISCUSSION THREAD

**Discussion by roy2100** [Johnserf-Seed/TikTokDownload#321] [Q&A]
本人维护了一版 go 版本的抖音下载 cli，功能很类似，最近用户反馈主页接口挂了。
https://github.com/nanlei2000/douyin_download
看了你这边的实现，需要通过接口获取  x-bogus 参数。
想问下，我可以在我的项目里使用你的生成 x-bogus 参数的接口吗？

**Comment by Johnserf-Seed** [comment]
> 本人维护了一版 go 版本的抖音下载 cli，功能很类似，最近用户反馈主页接口挂了。 https://github.com/nanlei2000/douyin_download 看了你这边的实现，需要通过接口获取 x-bogus 参数。 想问下，我可以在我的项目里使用你的生成 x-bogus 参数的接口吗？

xb已经开源了，随意使用 可自行部署
